src/puppetlabs/puppetdb/queue.clj

```diff
@@ -190,47 +198,65 @@
    :certname s/Str
    :producer-ts (s/maybe pls/Timestamp)
    :callback (s/=> s/Any s/Any)
-   :command-stream java.io.InputStream})
+   :command-stream java.io.InputStream
+   :compression s/Str})
 
 (s/defn create-command-req :- command-req-schema
   "Validating constructor function for command requests"
   [command :- s/Str
    version :- s/Int
    certname :- s/Str
    producer-ts :- (s/maybe s/Str)
+   compression :- s/Str
```

Comment:
I was just thinking through trying to minimize the number of places where code has to reference specific compression format values - in the event that more than one may exist in the future.  Could repurpose a enumerated type elsewhere too, though, so I'm good with going that way.